internal/runtimehandlerhooks/utils.go

```diff
@@ -121,3 +125,74 @@ func UpdateIRQSmpAffinityMask(cpus, current string, set bool) (cpuMask, bannedCP
 	}
 	return maskStringWithComma, invertedMaskStringWithComma, nil
 }
+
+func restartIrqBalanceService() error {
+	return exec.Command("service", "irqbalance", "restart").Run()
+}
+
+func updateIrqBalanceConfigFile(irqBalanceConfigFile, newIRQBalanceSetting string) error {
+	input, err := ioutil.ReadFile(irqBalanceConfigFile)
+	if err != nil {
+		return err
+	}
+	lines := strings.Split(string(input), "\n")
+	found := false
+	for i, line := range lines {
+		if strings.HasPrefix(line, irqBalanceBannedCpus+"=") {
+			lines[i] = irqBalanceBannedCpus + "=" + "\"" + newIRQBalanceSetting + "\""
+			found = true
```

Comment:
I think you can break after you find the needed line

Reply:
yes, done